IntentIq ID & Analytics Modules: adType and placementId parameters documentation

dmytro-po opened the pull request:
<!--
Thanks for improving the documentation 😃
Please give a short description and check the matching checkboxes to help us review this as quick as possible.

Please make the PR writeable. This allows us to fix typos, grammar and linting errors ourselves, which makes
merging and reviewing a lot faster for everybody.

⚠️ The documentation is merged after the related code changes are merged and release ⚠️
-->

## 🏷 Type of documentation
<!-- Remove items that don't apply and/or select an item by changing [ ] to [x] -->
- [x] new feature
- [x] new examples

## 📋 Checklist
<!-- Remove items that don't apply and/or select an item by changing [ ] to [x] -->
- [x] Related pull requests in prebid.js or server are linked -> https://github.com/prebid/Prebid.js/pull/12903

lpagnypubstack commented:
Hello @dmytro-po,
The array from "Function Parameters" on within file="dev-docs/analytics/intentiq.md" is not very well displayed : https://deploy-preview-5954--prebid-docs-preview.netlify.app/dev-docs/analytics/intentiq
Could you please add border to make it "cleaner" (same as row 36 from dev-docs/modules/userid-submodules/intentiq.md)?
`{: .table .table-bordered .table-striped }`
Thanks

dmytro-po commented:
> Hello @dmytro-po, The array from "Function Parameters" on within file="dev-docs/analytics/intentiq.md" is not very well displayed : https://deploy-preview-5954--prebid-docs-preview.netlify.app/dev-docs/analytics/intentiq Could you please add border to make it "cleaner" (same as row 36 from dev-docs/modules/userid-submodules/intentiq.md)? `{: .table .table-bordered .table-striped }` Thanks

Thank you! I've fixed the table and added the border as requested.
